app/controllers/application.js

```diff
@@ -186,4 +188,26 @@ export default class ApplicationController extends Controller.extend(
     const values = Object.values(state);
     return values.every(({ changed }) => changed === false);
   }
+
+  @action
+  toggleLeftSideMenuVisibility() {
+    this.leftSideMenuVisibilty = !this.leftSideMenuVisibilty;
+
+    const mapContainer = document.querySelector('.map-container');
+
+    if (this.leftSideMenuVisibilty)
+      mapContainer.setAttribute('class', 'map-container');
+    else mapContainer.setAttribute('class', 'map-container full-width');
+
```

Comment:
@dhochbaum-dcp I tried changing the class of the map container whenever the toggle is clicked here, but if you close the About section before toggling the sidebar the whitespace persists.